packages/web-app-files/src/components/Sidebar.vue

```diff
@@ -218,4 +222,7 @@ export default {
     fill: #ffba0a !important;
   }
 }
+#files-sidebar-item-name {
```

Comment:
Can be solved by giving `class="oc-text-initial"` to the h3 🤓

Reply:
Niceeee, was exactly looking for something like this! Thanks 👍